apps/settings/js/modules/wifi_utils.js

```diff
@@ -195,12 +194,14 @@ define(function(require) {
           }
         };
 
+        var key = WifiHelper.getKeyManagement(network);
         submitButton.disabled =
           !WifiHelper.isValidInput(key, password.value, identity.value,
             eap.value) || isSSIDInvalid();
       };
 
       eap.onchange = function() {
+        var key = WifiHelper.getKeyManagement(network);
```

Comment:
I guess my question is: would the `network` object will change within this function scope?